src/util/logging.go

```diff
@@ -0,0 +1,23 @@
+package util
```

Comment:
probably better to have a dedicated `log` package (or sub-package of `util`) so that you can reference `log.Error()` rather than `util.LogError()`, for example.